tests/unit/test_chainer.py

```diff
@@ -54,6 +63,8 @@ def sagemaker_session():
 
     describe = {'ModelArtifacts': {'S3ModelArtifacts': 's3://m/m.tar.gz'}}
     session.sagemaker_client.describe_training_job = Mock(return_value=describe)
+    session.sagemaker_client.describe_endpoint = Mock(return_value=ENDPOINT_DESC)
```

Comment:
Is it possible to make fixtures for these and autouse them?

Reply:
I tried make fixtures in conftest but then some tests would fail. For example test_estimator with different bucket name test fails.

Reply:
this is a fixture technically. she is doing the mock within the sagemaker_session fixture that is reused by other tests.